config/gnu-warnings/5

```diff
@@ -1,2 +1 @@
 -Warray-bounds=2
--Wc99-c11-compat
```

Comment:
Remove this warning since it doesn't make sense to warn about features missing from C99, but present in C11, when we're targeting C11 anyway.